language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -582,29 +602,27 @@ writePackageJson packageDir sdkVersion (Scope scope) depends =
    , "  \"version\": \"" <> version <> "\","
    , "  \"description\": \"Produced by daml2ts\","
    , "  \"dependencies\": {"
-   , "    \"@daml/types\": \"" <> version <> "\","
-   ] ++ dependencies ++
-   [ "    \"@mojotech/json-type-validation\": \"^3.1.0\""
-    , "  },"
+   ] ++ withCommas (staticDependencies ++ dependencies) ++
+   [ "  },"
     , "  \"scripts\": {"
     , "    \"build\": \"tsc --build\","
     , "    \"lint\": \"eslint --ext .ts src/ --max-warnings 0\""
     , "  },"
     , "  \"devDependencies\": {"
-    , "    \"@typescript-eslint/eslint-plugin\": \"^2.11.0\","
-    , "    \"@typescript-eslint/parser\": \"^2.11.0\","
-    , "    \"eslint\": \"^6.7.2\","
-    , "    \"typescript\": \"~3.7.3\""
-    , "  }"
-    , "}"
-    ])
+   ] ++ withCommas staticDevDependencies ++
```

Comment:
This whole block looks like it should really be using `aeson-pretty` instead of inlining a bunch of JSON in string literals.

Reply:
That is an awesome idea. Let's please do that. Perhaps in a PR _before_ this one.